app/views/posts/_form.html.slim

```diff
@@ -0,0 +1,33 @@
+= form_for Post.new, {remote:true}  do |f|
+
+	div class="modal-header"
+		button type="button" class="close" data-dismiss="modal" aria-label="Close" 
+			span aria-hidden="true" &times;
+		h3 class="modal-title" id="myModalLabel" 
+			b Add a Post
+	div class="modal-body" 
+		h4 what is it?
+		=f.text_area :short, maxlength: 140, placeholder: "your idea in 140 characters or less test" do
+		.counter 0
+		// Tagging section. See http://brianreavis.github.io/selectize.js/ 
+		h4 choose several tags
+		div.tag_container
+			= render partial: 'spaces/selectize_form', locals: {f:f, selected_tags: []}
+		 	
+		h4 more details:
+		=f.text_area :long, placeholder: "more information"
+		br 
+		br
+		div class="modal-footer"
+			button type="button" class="btn btn-default" data-dismiss="modal"Close
+			input type="submit" class='btn btn-primary' value="Add"
+
+coffee:
+	# "Selectize" the tag input
+	Spaces.selectizeElement '#post_tag_list'
+	# Attach callback to the post form
+	new Spaces.AjaxFormController '#addPostPopup', true, (data) -> 
+		Spaces.addPostCallback(data)
+	# Update counter
+	($ '#post_short').keyup (e) ->
+		($ '.counter').html ($ @).val().length 
```

Comment:
this is too much code in the markup - think about how to refactor this (along the lines of things I suggested above) to encapsulate the functionality for this screen inside a JS controller